feat: add `symbol/to-primitive` (#8494)
stdlib-js/stdlib: lib/node_modules/@stdlib/symbol/to-primitive/docs/repl.txt

{{alias}}
    toPrimitive symbol.

>     This symbol specifies a method that converts an object to a primitive value.

Reviewer: ```suggestion
    Symbol which specifies a method for converting an object to a primitive value.
```
